packages/story-editor/src/components/library/panes/media/common/mediaElement.js

```diff
@@ -203,7 +205,7 @@ function Element({
       onBlur={makeInactive}
       tabIndex={index === 0 ? 0 : -1}
     >
-      <InnerContainer>
+      <InnerContainer baseColor={backgroundColor}>
```

Comment:
Note sure, maybe want to use transient props here (`$baseColor={backgroundColor}`)